src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -263,11 +277,12 @@ export default {
 
 	created() {
 		const blurred = BrowserStorage.getItem('background-blurred')
-		if (blurred === null) {
-			BrowserStorage.setItem('background-blurred', 'true')
+		if (blurred === 'false' && isBackgroundBlurred === '') {
```

Comment:
maybe we should follow any value (`true` or `false`) that was set in browser storage as it is user's preference.

Reply:
`true` was a default value, so unlikely user even ticked that. If we force apply that to server, no one will get this enhancement.